Src/FluentAssertions/ObjectAssertionsExtensions.cs

```diff
@@ -112,6 +113,7 @@ public static class ObjectAssertionsExtensions
     /// <param name="becauseArgs">
     /// Zero or more objects to format using the placeholders in <paramref name="because" />.
     /// </param>
+    /// <exception cref="ArgumentNullException"><paramref name="options"/> is <c>null</c>.</exception>
```

Comment:
`assertions` should also be guard-checked ideally. (goes for any other extension method target like this)